doc/customizing_guide/viewer.rst

```diff
@@ -149,7 +149,7 @@ which should be used as follows::
 
         def __init__(self, *args, **kwargs):
             super(TutorialViewerState).__init__(*args, **kwargs)
-            MyExampleState.linestyle.set_choices(['solid', 'dashed', 'dotted'])
+            self.linestyle.set_choices(['solid', 'dashed', 'dotted'])
```

Comment:
```suggestion
            TutorialViewerState.linestyle.set_choices(['solid', 'dashed', 'dotted'])
```

Reply:
Yes, the proposed fix fails for me because `self.linestyle` is a just initialised `SelectionCallbackProperty()` here.
The equivalent in existing classes seems to be
https://github.com/glue-viz/glue/blob/f933d4bb69395c986ef1e2b3ec6ef0e64d422aea/glue/viewers/scatter/state.py#L319

Reply:
There seems to be another similar leftover further down at
https://github.com/glue-viz/glue-qt/blob/fe1868d13a9527fd3e2515c58ed58a19e79a0a19/doc/customizing_guide/viewer.rst?plain=1#L244